pandas/core/indexes/base.py

```diff
@@ -3818,11 +3899,11 @@ def _reindex_non_unique(self, target):
         return new_index, indexer, new_indexer
 
     _index_shared_docs['join'] = """
-        *this is an internal non-public method*
-
         Compute join_index and indexers to conform data
         structures to the new index.
 
+        *This is an internal non-public method*
```

Comment:
Not sure if this is actually true (didn't check where this is applied), but I'd remove this too.